syngenta_digital_dta/s3/adapter.py

```diff
@@ -55,7 +55,8 @@ def put(self, **kwargs):
             ACL=acl,
```

Comment:
we need to create another utility that wraps `put_object_tagging()` method for the future use